theano/ifelse.py

```diff
@@ -245,12 +246,14 @@ def thunk():
                     else:
                         for out, outtype, t in izip(outputs, outtypes, ts):
                             compute_map[out][0] = 1
+                            val = storage_map[t][0]
                             if self.as_view:
-                                oval = outtype.filter(storage_map[t][0])
+                                storage_map[out][0] = val
+                            # Work around broken numpy deepcopy
+                            elif type(a) in (numpy.ndarray, numpy.memmap):
```

Comment:
I don't want that. It add computation time (ifelse overhead is already big) and it do something bad for cdatatype (not copy them). Here we need a crash in case we can't deep copy it.